app/assets/javascripts/components/student_conference.es6.jsx

```diff
@@ -0,0 +1,46 @@
+class StudentConference extends Component {
+
+  static get propTypes() {
+    return {
+      student: React.PropTypes.object.isRequired,
+    };
+  }
+
+  static get defaultProps() {
+    return {
+      student: {},
+    };
+  }
+
+  get styles() {
+    return {
+      container: {
+        display: 'flex',
+        flexDirection: 'column',
+        flex: '1',
+        position: 'relative',
+        padding: '12px',
+      },
+      heading: {
+        fontSize: '16px'
+      }
+    };
+  }
+
+  render() {
+    var student = this.props.student;
+    return (
+      <div style={this.styles.container}>
+        <span style={this.styles.heading}>{'Status'}</span>
+          {student.status + '\n\n'}
```

Comment:
Instead of using `'\n\n`, it's probably better to just set the style of `styles.heading` to `display: block` or to change `span` to `h2` or `h3`.